webpack.config.js

```diff
@@ -0,0 +1,19 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = {
+  mode: 'development',
```

Comment:
Just decided to go with `development` for now https://webpack.js.org/configuration/mode/